about.html

```diff
@@ -8,58 +8,97 @@
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
-
+  <link rel="stylesheet" href="css.About.css">
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
 
 
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
 
-    Learn More
+<!-- <div class="static-navbar"> -->
+   
+    <div class='wrapper'> 
+        
 
+    <nav>
+        <div>
+            <img class="logo" src="img/lambda-black.png" alt="Lambda logo.">
+        </div>  
+      <a href="#">Home</a>
+        <a href="about.html">About</a>
+        <a href="#">Products</a>
+        <a href="#">Blog</a>
+        <a href="#">Contact</a>
+        
```

Comment:
you should have the same nav code as the index page to keep the pages consistent.  this nav bar is different than the home page nav bar, so the styles don't match up when changing pages.